monai/transforms/spatial/array.py

```diff
@@ -12,11 +12,15 @@
 A collection of "vanilla" transforms for spatial operations
 https://github.com/Project-MONAI/MONAI/wiki/MONAI_Design
 """
+
+from __future__ import annotations
+
 import warnings
+from collections.abc import Callable
 from copy import deepcopy
 from enum import Enum
 from itertools import zip_longest
-from typing import Any, Callable, Dict, List, Optional, Sequence, Tuple, Union, cast
+from typing import Any, Optional, Sequence, Tuple, Union, cast
```

Comment:
I think we still need the deprecated types here, since the new ones only work inside annotations, but this is creating a type alias.
```python
RandRange = Optional[Union[Sequence[Union[Tuple[float, float], float]], float]]
```